packages/contact-center/store/src/storeEventsWrapper.ts

```diff
@@ -297,7 +297,7 @@ class StoreWrapper implements IStoreWrapper {
     const taskToRemove = this.store.taskList.find((task) => task.data.interactionId === taskId);
     if (taskToRemove) {
       taskToRemove.off(TASK_EVENTS.TASK_ASSIGNED, this.handleTaskAssigned);
-      taskToRemove.off(TASK_EVENTS.TASK_END, ({wrapupRequired}) => this.handleTaskEnd(taskToRemove, wrapupRequired));
+      taskToRemove.off(TASK_EVENTS.TASK_END, () => this.handleTaskEnd(taskToRemove));
```

Comment:
Shouldn't we rather be passing the payload?
```suggestion
      taskToRemove.off(TASK_EVENTS.TASK_END, this.handleTaskEnd);
```